frontend/src/components/core/ReportView/ReportView.scss

```diff
@@ -77,19 +77,6 @@
     }
   }
 
-  &.--with-sidebar {
```

Comment:
Should we remove this class from `ReportView.tsx` since it's no longer used? 
https://github.com/streamlit/streamlit/blob/develop/frontend/src/components/core/ReportView/ReportView.tsx#L77

Reply:
Good catch, thanks